docs_src/use-cases/template.md

```diff
@@ -0,0 +1,78 @@
+## Use case template <!-- Replace with use case title -->
+This is a template for EdgeX use cases. The description of each section is included as comments and visible only when viewing the source. Remove this paragraph before submitting a use case.
+
+### Submitter(s)
+<!-- List use case submitters
+
+Format:
+- Name (Organization)
+-->
+
+### Status
+<!-- Status of the submission and date. 
+Status is one of: accepted, amended, deprecated
+Date is ISO 8601 (YYYY-MM-DD)
+
+E.g.:
+- [accepted](URL of PR) (2022-04-01)
+- [amended](URL of PR) (2022-05-01)
+-->
```

Comment:
I thought about adding a "supported" state, as described in #695. But I think it is redundant and not appropriate for two reasons:
1. Requirements and corresponding feature requests will indicate the support and implementation
2. It won't allow setting support for parts of the use case. It is all or none.

I did not include a "superseded" state, in favor of the "amended" one. We either amend a useful document or discard completely, i.e. "deprecated"